tests/test_metadata.py

```diff
@@ -263,7 +263,7 @@ def preserve_track_obsolete_file_flag():
 
 
 def test_track_obsolete_files_defaults_to_true(preserve_track_obsolete_file_flag):
```

Comment:
nit: should probably rename the test too

Reply:
Done.